metapool/sample_sheet.py

```diff
@@ -254,6 +255,47 @@ def _is_empty(csv_line):
                     section[key] = value
                     continue
 
+        # For known popular alternatives of fields in [Data] section,
+        # proactively convert them to standard format.
+        alts = {'well_description': 'Well_description',
+                'description': 'Well_description',
+                'Description': 'Well_description',
+                'sample_plate': 'Sample_Plate'}
+
+        # assume all samples represent an identical list and order of columns.
+        # alternatives found in first sample will be true for all other
+        # samples as well.
+        if not self._samples:
+            # If this object has no samples, skip renaming of columns in [Data]
+            # section.
+            return
+
+        warning_msgs = []
+
+        obs = self._samples[0]._store
```

Comment:
This is interacting with a private member variable of another object defined in an external library. This is not safe to do. Why not modify `section_header` above?

Reply:
doh, it pays to look a little higher in the code, it seems. Thanks!

It is true, it's not safe to do, but at the time it looked like what needed to be done w/out modifying sample_sheets library directly.